packages/editor/src/core/editor.tsx

```diff
@@ -135,8 +140,8 @@ function InnerDocument({ children, onChange, ...props }: EditorProps) {
   function renderChildren(id: string) {
     const document = <SubDocument id={id} />
 
-    if (typeof children === 'function') {
-      return children(document)
+    if (typeof children === 'function' && loggedInData) {
```

Comment:
This seemed like a possible bug trigger, but I didn't notice any problems while testing.